core/shared/src/main/scala/zio/prelude/IdentityBoth.scala

```diff
@@ -64,6 +64,15 @@ object IdentityBoth extends LawfulF.Invariant[DeriveEqualIdentityBothInvariant,
   def apply[F[_]](implicit identityBoth: IdentityBoth[F]): IdentityBoth[F] =
     identityBoth
 
+  /**
+   * The `IdentityBoth` instance for `ZIO`.
+   */
+  implicit def ZIOIdentityBoth[R, E]: IdentityBoth[({ type lambda[+a] = ZIO[R, E, a] })#lambda] =
```

Comment:
Actually, could you do it the other way around? :pray: 
The proper way is to have `ZIOIdentityBoth` in `AssociativeBoth`'s companion object.

Reply:
moved